snap/snapcraft.yaml

```diff
@@ -82,7 +82,11 @@ parts:
   device-modbus:
     source: .
     plugin: make
-    build-packages: [git]
+    build-packages:
```

Comment:
ASC has this also:
https://github.com/edgexfoundry/app-service-configurable/blob/master/snap/snapcraft.yaml#L75
@tonyespy should this snap also have this?

Reply:
@lenny-intel yes